OpenRA.Platforms.Default/OpenAlSoundEngine.cs

```diff
@@ -311,6 +311,26 @@ public void SetListenerPosition(WPos position)
 			AL10.alListenerf(EFX.AL_METERS_PER_UNIT, .01f);
 		}
 
+		public void ReleaseSourcePool()
+		{
+			foreach (var slot in sourcePool)
+				if (slot.Value.Sound != null)
+					ReleaseSound(slot.Key);
+		}
+
+		public void ReleaseSound(ISound sound)
+		{
+			var openAlSound = sound as OpenAlSound;
+			if (openAlSound != null)
+				ReleaseSound(openAlSound.Source);
+		}
+
+		void ReleaseSound(uint source)
+		{
+			AL10.alSourceStop(source);
+			AL10.alSourcei(source, AL10.AL_BUFFER, 0);
```

Comment:
As I read the docs, this breaks the link between the sound and the buffer, but the buffer itself remains alive? If that is true, this doesn't really release anything. Your newly added `Dispose` on `OpenAlSoundSource` would actually be freeing the buffers.

If that's true, you could basically delete these new functions, and where you call them just use `StopSound` or `StopAudio` if you wanted to just ensure audio was stopped.

If this has another side-effect I'm not aware of, please let me know (maybe add a comment?).